docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
+----
+
+* Replace `<source_node>` with the hostname of the source node.
+
+The source node is not part of the cluster anymore, and should not appear in
+the {Ceph} host list when running `cephadm shell -- ceph orch host ls`.
+However, a `sudo podman ps` in the source node might list both mon and mgr still
+running.
+
+----
+[root@oc0-controller-1 ~]# sudo podman ps
+CONTAINER ID  IMAGE                                                                                        COMMAND               CREATED         STATUS             PORTS       NAMES
+ifeval::["{build}" != "downstream"]
+5c1ad36472bc  quay.io/ceph/daemon@sha256:320c364dcc8fc8120e2a42f54eb39ecdba12401a2546763b7bef15b02ce93bc4  -n mon.oc0-contro...  35 minutes ago  Up 35 minutes ago              ceph-f6ec3ebe-26f7-56c8-985d-eb974e8e08e3-mon-oc0-controller-1
+3b14cc7bf4dd  quay.io/ceph/daemon@sha256:320c364dcc8fc8120e2a42f54eb39ecdba12401a2546763b7bef15b02ce93bc4  -n mgr.oc0-contro...  35 minutes ago  Up 35 minutes ago              ceph-f6ec3ebe-26f7-56c8-985d-eb974e8e08e3-mgr-oc0-controller-1-mtxohd
+endif::[]
+ifeval::["{build}" == "downstream"]
+5c1ad36472bc  registry.redhat.io/ceph/rhceph@sha256:320c364dcc8fc8120e2a42f54eb39ecdba12401a2546763b7bef15b02ce93bc4  -n mon.oc0-contro...  35 minutes ago  Up 35 minutes ago              ceph-f6ec3ebe-26f7-56c8-985d-eb974e8e08e3-mon-oc0-controller-1
+3b14cc7bf4dd  registry.redhat.io/ceph/rhceph@sha256:320c364dcc8fc8120e2a42f54eb39ecdba12401a2546763b7bef15b02ce93bc4  -n mgr.oc0-contro...  35 minutes ago  Up 35 minutes ago              ceph-f6ec3ebe-26f7-56c8-985d-eb974e8e08e3-mgr-oc0-controller-1-mtxohd
+endif::[]
+----
+
+To cleanup the source node before moving to the next phase, cleanup the existing
+containers and remove the cephadm related data from the node.
+// fpantano: there's an automated procedure run through cephadm but it's too
+// risky. If the user doesn't perform it properly the cluster can be affected.
+// We can put a downstream comment to contact the RH support to clean the source
+// node up in case of leftovers, and open a bug for cephadm.
+
+//. ssh into one of the existing Ceph mons (usually controller-1 or controller-2)
+. Prepare the target node to host the new mon and add the `mon` label to the
+target node:
++
+----
+for label in mon mgr _admin; do
+    ceph orch host label add <target_node> $label; done
+done
+----
+
+* Replace <target_node> with the hostname of the host listed in the {CephCluster}
+  through the `ceph orch host ls` command.
+
+At this point the cluster is running with only two mons, but a third mon appears
+and will be deployed on the target node.
+However, The third mon might be deployed on a different ip address available in
+the node, and you need to redeploy it when the ip migration is concluded.
+Even though the mon is deployed on the wrong ip address, it's useful keep the
+quorum to three and it ensures we do not risk to lose the cluster because two
+mons go in split brain.
```

Comment:
This text looks like it should be a note.

Reply:
Added the note header